sdk/cosmosdb/cosmos/src/client/SasToken/SasTokenProperties.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { SasTokenPartitionKeyValueRange } from "./SasTokenPartitionKeyValueRange";
+import { SasTokenPermissionKind } from "../../client/SasToken/SasTokenPermissionKind";
+import { CosmosContainerChildResourceKind } from './CosmosContainerChildResourceKind';
+import { CosmosKeyType } from './CosmosKeyType';
+
+export interface SasTokenProperties {
```

Comment:
Remove this interface. In general its not idiomatic in TypeScript to separate out an interface if its only used a single time. This is only implemented once in `SasTokenPartitionKeyValueRange`